airflow/config_templates/default_airflow.cfg

```diff
@@ -718,6 +722,17 @@ tolerations =
 # The worker pods will be scheduled to the nodes of the specified key-value pairs.
 # Should be supplied in the format: key = value
 
+[kubernetes_environment_variables]
```

Comment:
Can we move this section under [kubernetes_annotations] ? Since the functionality is similar to the "setting environment variables through secrets

Also if the docs for those two sections and examples could follow the same structure